[Bug]: Publishing of docker images is broken

### What happened?

Operator docker images are not published on dockerhub. E.g. image for 1.48.0 release https://hub.docker.com/repository/docker/jaegertracing/jaeger-operator/general

Failed job: https://github.com/jaegertracing/jaeger-operator/actions/runs/6013009171/job/16311179997


```
#18 [linux/ppc64le stage-1 2/5] RUN INSTALL_PKGS="openssl" &&     yum install -y $INSTALL_PKGS &&     rpm -V $INSTALL_PKGS &&     yum clean all &&     mkdir /tmp/_working_dir &&     chmod og+w /tmp/_working_dir
#18 ERROR: process "/bin/sh -c INSTALL_PKGS=\"openssl\" &&     yum install -y $INSTALL_PKGS &&     rpm -V $INSTALL_PKGS &&     yum clean all &&     mkdir /tmp/_working_dir &&     chmod og+w /tmp/_working_dir" did not complete successfully: exit code: 1

#20 [linux/amd64 builder  3/14] COPY hack/install/install-dependencies.sh hack/install/
#20 CANCELED

#16 [linux/arm64 stage-1 2/5] RUN INSTALL_PKGS="openssl" &&     yum install -y $INSTALL_PKGS &&     rpm -V $INSTALL_PKGS &&     yum clean all &&     mkdir /tmp/_working_dir &&     chmod og+w /tmp/_working_dir
#16 13.07 Updating Subscription Management repositories.
#16 13.08 Unable to read consumer identity
#16 13.08 Subscription Manager is operating in container mode.
#16 13.24 
#16 13.24 This system is not registered with an entitlement server. You can use subscription-manager to register.
#16 13.24 
#16 16.33 Red Hat Universal Base Image 8 (RPMs) - BaseOS  367 kB/s | 580 kB     00:01    
#16 CANCELED

#15 [linux/s390x stage-1 2/5] RUN INSTALL_PKGS="openssl" &&     yum install -y $INSTALL_PKGS &&     rpm -V $INSTALL_PKGS &&     yum clean all &&     mkdir /tmp/_working_dir &&     chmod og+w /tmp/_working_dir
#15 17.12 Red Hat Universal Base Image 8 (RPMs) - BaseOS  290 kB/s | 586 kB     00:02    
#15 CANCELED
------
 > [linux/ppc64le stage-1 2/5] RUN INSTALL_PKGS="openssl" &&     yum install -y $INSTALL_PKGS &&     rpm -V $INSTALL_PKGS &&     yum clean all &&     mkdir /tmp/_working_dir &&     chmod og+w /tmp/_working_dir:
7.468 warning: rpmdb: BDB2053 Freeing read locks for locker 0x11: 2222/140735876952432
7.468 warning: rpmdb: BDB2053 Freeing read locks for locker 0x12: 2222/140735876952432
7.468 warning: rpmdb: BDB2053 Freeing read locks for locker 0x13: 2222/140735876952432
7.468 warning: rpmdb: BDB2053 Freeing read locks for locker 0x14: 2222/140735876952432
13.35 Updating Subscription Management repositories.
13.36 Unable to read consumer identity
13.50 
13.50 This system is not registered with an entitlement server. You can use subscription-manager to register.
13.50 
13.64 Error: There are no enabled repositories in "/etc/yum.repos.d", "/etc/yum/repos.d", "/etc/distro.repos.d".
------
Dockerfile:42
--------------------
  41 |     
  42 | >>> RUN INSTALL_PKGS="openssl" && \
  43 | >>>     yum install -y $INSTALL_PKGS && \
  44 | >>>     rpm -V $INSTALL_PKGS && \
  45 | >>>     yum clean all && \
  46 | >>>     mkdir /tmp/_working_dir && \
  47 | >>>     chmod og+w /tmp/_working_dir
  48 |     
--------------------
ERROR: failed to solve: process "/bin/sh -c INSTALL_PKGS=\"openssl\" &&     yum install -y $INSTALL_PKGS &&     rpm -V $INSTALL_PKGS &&     yum clean all &&     mkdir /tmp/_working_dir &&     chmod og+w /tmp/_working_dir" did not complete successfully: exit code: 1
make: *** [Makefile:156: dockerx] Error 1
Error: Process completed with exit code 2.
```

### Steps to reproduce

Run publish docker image GHA

### Expected behavior

Docker images should be published 

### Relevant log output

_No response_

### Screenshot

_No response_

### Additional context

_No response_

### Jaeger backend version

_No response_

### SDK

_No response_

### Pipeline

_No response_

### Stogage backend

_No response_

### Operating system

_No response_

### Deployment model

_No response_

### Deployment configs

_No response_

--- a/Dockerfile
+++ b/Dockerfile
@@ -34,15 +34,14 @@ ARG TARGETARCH
 # Build
 RUN CGO_ENABLED=0 GOOS=linux GOARCH=${TARGETARCH} GO111MODULE=on go build -ldflags="-X ${VERSION_PKG}.version=${VERSION} -X ${VERSION_PKG}.buildDate=${VERSION_DATE} -X ${VERSION_PKG}.defaultJaeger=${JAEGER_VERSION}" -a -o jaeger-operator main.go
 
-FROM registry.access.redhat.com/ubi8/ubi
+FROM ubuntu:22.04
 
 ENV USER_UID=1001 \
     USER_NAME=jaeger-operator
 
 RUN INSTALL_PKGS="openssl" && \
-    yum install -y $INSTALL_PKGS && \
-    rpm -V $INSTALL_PKGS && \
-    yum clean all && \
+    apt update && \
+    apt install -y $INSTALL_PKGS && \
     mkdir /tmp/_working_dir && \
     chmod og+w /tmp/_working_dir
 

--- a/Dockerfile
+++ b/Dockerfile
@@ -34,14 +34,15 @@ ARG TARGETARCH
 # Build
 RUN CGO_ENABLED=0 GOOS=linux GOARCH=${TARGETARCH} GO111MODULE=on go build -ldflags="-X ${VERSION_PKG}.version=${VERSION} -X ${VERSION_PKG}.buildDate=${VERSION_DATE} -X ${VERSION_PKG}.defaultJaeger=${JAEGER_VERSION}" -a -o jaeger-operator main.go
 
-FROM ubuntu:22.04
+FROM quay.io/centos/centos:stream9
 
 ENV USER_UID=1001 \
     USER_NAME=jaeger-operator
 
 RUN INSTALL_PKGS="openssl" && \
-    apt update && \
-    apt install -y $INSTALL_PKGS && \
+    dnf install -y $INSTALL_PKGS && \
+    rpm -V $INSTALL_PKGS && \
+    dnf clean all && \
     mkdir /tmp/_working_dir && \
     chmod og+w /tmp/_working_dir
 

